src/main/features/claimant-response/tasks/freeMediationTask.ts

```diff
@@ -1,17 +1,10 @@
-import { Validator } from '@hmcts/class-validator'
-
-import { FeatureToggles } from 'utils/featureToggles'
 import { MediationDraft } from 'mediation/draft/mediationDraft'
-import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
-
-const validator = new Validator()
+import { FreeMediationTask } from 'shared/components/free-mediation/freeMediationTask'
 
-export class FreeMediationTask {
-  static isCompleted (claimantResponseDraft: DraftClaimantResponse, mediationDraft: MediationDraft): boolean {
-    if (FeatureToggles.isEnabled('mediation')) {
-      return mediationDraft.willYouTryMediation !== undefined && validator.validateSync(mediationDraft.willYouTryMediation).length === 0
-    } else {
-      return claimantResponseDraft.freeMediation !== undefined && validator.validateSync(claimantResponseDraft.freeMediation).length === 0
-    }
+export class ClaimantResponseFreeMediationTask extends FreeMediationTask {
+  static isCompleted (mediationDraft: MediationDraft): boolean {
+    return (this.isWillYouTryMediationCompleted(mediationDraft)) ||
+      this.isYouCanOnlyUseMediationCompleted(mediationDraft) &&
+      this.isCanWeUseCompleted(mediationDraft)
```

Comment:
I think this is correct, but adding parentheses to clarify could be helpful.
```
     A ||  B  && C
===  A || (B  && C)
!== (A ||  B) && C
```

Reply:
i have added parentheses for it to make more sense

Reply:
done